src/domain/configure_ftp_cameras.py

```diff
@@ -252,17 +265,11 @@ def add_ftp_camera(self):
     def test_ftp_server(self):
         """Method to test out FTP server configuration options by running a FTP server instance."""
 
-        logger.info("Starting test FTP server...")
-        FTPsServer.ftps_server = FTPsServer(self.ui.lineEdit_ip.text(),
-                                           int(self.ui.lineEdit_port.text()),
-                                           int(self.ui.lineEdit_max_conn.text()),
-                                           int(self.ui.lineEdit_max_conn_ip.text()),
-                                           self.ui.label_certificate_file_path.text(),
-                                           self.ui.label_key_file_path.text(),
+        # TODO: iterate over all users on UI
+        if not FTPsServer.ftps_server.has_user(self.ui.lineEdit_username_1.text()):
+            FTPsServer.ftps_server.add_user(self.ui.lineEdit_username_1.text(),
```

Comment:
if not FTPsServer.ftps_server.has_user(username := self.ui.lineEdit_username_1.text()):
            FTPsServer.ftps_server.add_user(username,

Reply:
updated code